README.md

```diff
@@ -49,6 +50,9 @@ Packages include:
   - Various vector drawing methods for drawing paths, polygons etc.
   - Text drawing.
 
+- **SixLabors.ImageSharp.Web**
```

Comment:
(as above)
ImageSharp.Web is in a separate [respository](https://github.com/SixLabors/ImageSharp.Web) so shouldn't really be included in this readme